packages/react/src/progress-indicator/docs/overview.mdx

```diff
@@ -126,59 +127,72 @@ The `status` prop can be used to indicate the status of each item. The following
 	</TableBody>
 </Table>
 
-## Active state handling
+## Currently active item
 
-The `isActive: true` property needs to be applied to the currently active item.
+To indicate the currently active item, whether level 1 or level 2, pass in a matching path to the `activePath` prop. E.g. `activePath="/organisations/change-name"`.
 
-Currently, only the `'started'`, and `'blocked'` statuses may receive the `isActive: true` property.
-
-If the user navigates to a page that is a different status, the status of that page should be changed to `'started'` (unless `'blocked'` is more appropriate).
-
-## Buttons
+## Hiding the sub title
 
-If an item does not specify an `href` attribute, a `button` element will be rendered. This enables you to respond to various mouse events, such as `onClick`.
+The Progress indicator component displays a subtitle "x of y steps completed" by default. To hide this subtitle, set the `hideSubtitle` prop to `true`.
 
 ```jsx live
 <ProgressIndicator
+	activePath="/organisations"
```

Comment:
Can we make `activePath` optional?

If it is undefined, attempt to determine the `activePath` via the url. If it is explicitly defined then use that value to determine which href is active.

Reply:
This would make things inconsistent with our other `activePath` components.

Reply:
I feel like this is another ticket to do this for all components with `activePath` if we decide it has value